tasks/main.yml

```diff
@@ -16,13 +16,24 @@
     group: "{{ deploydrupal_checkout_user }}"
     mode: ug+w
 
-- name: open permissions on settings file.
+- name: discover settings files.
+  find:
+    paths: "{{ deploydrupal_core_path }}/sites"
```

Comment:
Some of our more recent sites leverage `sites/default` for shared settings files and `sites/SITENAME` for site specific files. This should accommodate that need as well.

Perhaps that should be added as a comment?

Reply:
I'd suggest instead of a comment, amending the task's `name` property so it'll show in the _output_ as well.